### PR TITLE
Add Building Microservices architecture with ASP.NET to Courses

### PR DESCRIPTION
**Pull request by Fazer01**
## What does this PR do?
Add course about Building a microservice architecture in ASP.NET by Gill Cleeren
### Description
Explore a sample microservice architecture fully based on ASP.NET Core 6 highlighting many new features and building blocks.

### Why is this valuable (or not)?
Since there was not already a course with this subject I thought it might be valuable to add it.

### How do we know it's really free?
It's on youtube. 

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -242,6 +242,7 @@
 
 ### <a id="csharp"></a>C\#
 
+* [Building a microservice architecture with ASP.NET Core](https://www.youtube.com/watch?v=SR53SKIUYPA&list=PL03Lrmd9CiGfgBmqFWeRxaUSQ_cSF_rCF&index=30) - Gill Cleeren (Youtube)
```
Comment (eshellman) [inline courses/free-courses-en.md:245]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md:245]: As requested by Eric...

```suggestion
* [Building a microservice architecture with ASP.NET Core - Gill Cleeren - NDC London 2022](https://www.youtube.com/watch?v=SR53SKIUYPA) - Gill Cleeren
```

**Comment by Fazer01** [comment]
Provided changes as requested @davorpa

**Comment by Fazer01** [comment]
Provided changes as requested @davorpa

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**